python/tank/bootstrap/async.py

```diff
@@ -0,0 +1,278 @@
+# Copyright (c) 2016 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+# Import Qt without having to worry about the version to use.
+from ..util.qt_importer import QtImporter
```

Comment:
@manneohrstrom, @jfboismenu, I see a problem in [here](https://github.com/shotgunsoftware/tk-core/blob/master/python/tank/authentication/ui/qt_abstraction.py) since calling `QtImporter()` does not raise an `ImportError` exception when no Qt is available, but simply set its `self.QtCore`, `self.QtGui` and `self.wrapper` to `None`.